kura/org.eclipse.kura.net.configuration/src/main/java/org/eclipse/kura/net/configuration/NetworkConfigurationPropertyNames.java

```diff
@@ -22,8 +22,15 @@ public enum NetworkConfigurationPropertyNames {
     CONFIG_IPV4_ADDRESS,
     CONFIG_IPV4_PREFIX,
     CONFIG_IPV4_GATEWAY,
-    CONFIG_DNS_SERVERS,
```

Comment:
Isn't this a breaking change? @pierantoniomerlino

Reply:
@MMaiero The package is exported as internal. Is it still a breaking change?

Reply:
if it has implications to the snapshot, probably it is a problem, right?

Reply:
AFAIK, this enum is used only to retrieve the Messages in the `/resources /.../NetworkConfigurationMessagesBundle.properties`, so I think that we can change it. I'd to admit that the name of the enum is a bit misleading.

Reply:
ok for me if we have the confidence that this is not going to be an issue for users that will use old snapshots in this new version